read_files.py

```diff
@@ -0,0 +1,15 @@
+import os
```

Comment:
This is the solution to a problem that I got by email.
If a file/script is alone and it solves one problem, it is a common
practice to put that at the top as a comment.
In pyhton it can be done by a muti line comment.

Reply:
understood!